fluentd-gcp-image/run.sh

```diff
@@ -17,13 +17,4 @@
 # These steps must be executed once the host /var and /lib volumes have
 # been mounted, and therefore cannot be done in the docker build stage.
 
-# For systems without journald
-mkdir -p /var/log/journal
-
-# Copy host libsystemd into image to avoid compatibility issues.
-if [ ! -z "$(ls /host/lib/libsystemd* 2>/dev/null)" ]; then
-  rm /lib/x86_64-linux-gnu/libsystemd*
-  cp -a /host/lib/libsystemd* /lib/x86_64-linux-gnu/
-fi
-
-/usr/local/bin/fluentd $@
+exec /usr/local/bin/fluentd "$@"
```

Comment:
Can we delete this script now?

Reply:
As I pointed out in the PR/commit description, I suggest keeping it around just in case someone is still running it with something like:

```shell
docker run gcr.io/google-containers/fluentd-gcp:latest /run.sh $FLUENTD_ARGS
```

I don't think it's that detrimental to keep it around, since the space it takes is really minimal...

So I think I'd rather not delete it, or at least not delete it right away. I could be swayed the other way.

Reply:
Oh sorry, I missed that.  I suggest adding this to the script, so we can clean it up later (and it's clear why it's still here):

```shell
echo >&2 "DEPRECATED: use /usr/local/bin/fluentd directly"
```